lib/Service/ConfigService.php

```diff
@@ -124,6 +130,17 @@ public function isCardDetailsInModal(int $boardId = null): bool {
 		return (bool)$this->config->getUserValue($this->getUserId(), Application::APP_ID, 'board:' . $boardId . ':cardDetailsInModal', $defaultState);
 	}
 
+	public function isCardIdBadgeEnabled(int $boardId  = null): bool 
+	{
+		if ($this->getUserId() === null) {
+			return false;
```

Comment:
This may not be used anyways, but we could take the `$appConfigState` so the admin can set this in case we introduce public shares.

Reply:
@juliushaertl I am unsure of what you need me to change when you say "take the `$appConfigState`". Do you have an example?

Reply:
@juliushaertl it is the same procedure is the other methods. It can be written so, in my opinion

Reply:
@juliushaertl @ultrasites either way, if someone could explain what should be changed, I'm happy to do so, otherwise pls resolve this.